registration/scripts/directives.js

```diff
@@ -56,7 +63,13 @@
                             if (angular.isDefined(deferred)) {
                                 deferred.notify('reject', response);
                             }
+                            apiError = true;
                         });
+                    } else {
+                        //check there are no other error messages on the screen from the api
+                        if (!apiError) {
+                            scope.emptyEmail = true;
+                        }
```

Comment:
It seems sensible to trust that the API is validating on blank emails (especially as this library is built around `django-user-management`), rather than having that edge case validated on the front-end. We're not doing any other validation here.